Assets/MixedRealityToolkit/_Core/Interfaces/IMixedRealityDevice.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions
+{
+    public interface IMixedRealityDevice
+    {
+        void Initialize();
```

Comment:
We seem to use these interface methods a lot, should we make a base base interface for basic methods like this we keep reusing?

ie. Initialize, Enable, Disable, Destory, etc?

Reply:
Will review once we get further.